x-pack/plugins/cloud_security_posture/public/types.ts

```diff
@@ -24,5 +25,6 @@ export interface CspClientPluginStartDeps {
   // required
   data: DataPublicPluginStart;
   unifiedSearch: UnifiedSearchPublicPluginStart;
+  charts: ChartsPluginStart;
```

Comment:
no `ChartsPluginSetup` ?

Reply:
as far as i saw from other usages seems like setup is not used